scripts/run.sh

```diff
@@ -72,15 +72,15 @@ elif [ $1 == "test" ]; then
         echo
 
         if [[ $backend_rc == 0 && $frontend_rc == 0 ]]; then
-            echo "All tests pass!!!"
-            if [[ $GITHUB_HEAD_REPO_URL == "https://github.com/mozilla/balrog.git" ]];
+            if [ '$GITHUB_HEAD_REPO_URL' == "https://github.com/mozilla/balrog.git" ];
```

Comment:
Hey, do you think we can use another variable as a condition? Probably we define a variable in the taskcluster environment and check for that to know whether to send coverage or not.